index.html

```diff
@@ -31,9 +31,144 @@ <h1 class="introduction"> Hey there. I’m Chukwuemeka <br> <span> I’m a softw
 
                  </ul>
         </footer>
+    </section>
+
+<!-- My Recent Work Section      -->
+
+        <section id="myrecentwork" class="grid-container">
+            <h2 class="recentwork"> My Recent<br> Works</h2>
```

Comment:
- [OPTIONAL] Please follow the style of the provided template and create the items before and after the text. Will create a better UI. As not required by the previous reviewer I consider it optional.

Figma template
<img width="372" alt="Screen Shot 2022-12-01 at 11 29 24 PM" src="https://user-images.githubusercontent.com/47673311/205202352-a61703ab-fc47-42b3-907d-c11e4b035800.png">

Your design
<img width="372" alt="Screen Shot 2022-12-01 at 11 27 33 PM" src="https://user-images.githubusercontent.com/47673311/205202395-8c81ac28-445e-477f-a0c9-e34a13814d1f.png">